.github/workflows/pre-commit.yaml

```
@@ -0,0 +1,72 @@
+name: Pre-commit
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)